example/ios/Podfile.lock

```diff
@@ -654,4 +654,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 5eddd6af0376deacea0a9cf5568699be94aab722
 
-COCOAPODS: 1.11.3
+COCOAPODS: 1.12.0
```

Comment:
If we upgrade version for cocoapods, Everyone have to upgrade too. it may effect to their current project.

Reply:
updated